packages/react-tinacms-github/src/github-client/github-client.ts

```diff
@@ -120,12 +125,11 @@ export class GithubClient {
   get workingRepoFullName(): string {
     const forkName = this.getCookie(GithubClient.WORKING_REPO_COOKIE_KEY)
 
-    if (!forkName) {
-      // TODO: Right now the client only works with forks. This should go away once it works with origin.
-      throw new Error('GithubClient cannot find name of fork')
+    if (forkName) {
+      return forkName
     }
 
-    return forkName
+    return this.baseRepoFullName
```

Comment:
Looks good, my one worry is that baseRepo is used as a fallback instead of needing to be explicitly set as the working Repo within the cookie. 

For instance, if your fork cookie was to expire, or get deleted, we would switch you to the baseRepo which might not be ideal if you're about to save.

Reply:
Just added the part that sets the cookie.

I don't think the cookie expiring will be a problem though. This change prevents you from ever working from a fork if you have permission to do so.

Reply:
> I don't think the cookie expiring will be a problem though. This change prevents you from ever working from a fork if you have permission to do so.

Even if I had permission to edit on the base repo, I think I'd still want to have to explicitly switch my working repo to the base repo, rather than have it switch automatically if my WORKING_REPO_COOKIE expired.